src/server/pmix_server_get.c

```diff
@@ -531,7 +531,9 @@ static pmix_status_t _satisfy_request(pmix_nspace_t *nptr, pmix_rank_t rank,
      * the corresponding GDS to retrieve the data. Otherwise,
      * the data will have been stored under our GDS */
     if (0 < nptr->nlocalprocs) {
-        *local = true;
+        if (local) {
```

Comment:
There is here: [pmix_server_get.c:271](https://github.com/pmix/pmix/blob/ef707ab63511bbe38bdb684d09e8cbb5ff63cf17/src/server/pmix_server_get.c#L721)

```C
pmix_status_t rc;
    rc = _satisfy_request(nptr, rank, scd, req->cbfunc, req->cbdata, NULL);
    if( PMIX_SUCCESS != rc ){
```